molecule/multimachine/hosts.yml

```diff
@@ -1,10 +1 @@
-all:
-  children:
-    external:
-      children:
-        external_activemq:
-          hosts:
-            external_activemq_hostname:
-              activemq_username: alfresco
-              activemq_port: 61617
-              activemq_protocol: ssl
+# placeholder
```

Comment:
we may want to remove and gitignore this file to avoid unintentional commit?

Reply:
I kept a placeholder as the file is referenced in the molecule config and its absence breaks the molecule run with:
```shell
CRITICAL The source path '/Users/Alexandre.Chapellon@hyland.com/workspace/alfresco-ansible-deployment/molecule/multimachine/./hosts.yml' does not exist.
```

Reply:
Oh just got it now 👍